blockchains/smartchain/assets/0x26619FA1d4c957C58096bBbeCa6588dCFB12E109/info.json

```diff
@@ -0,0 +1,28 @@
+{
+    "name": "TIME",
+    "type": "BEP20",
+    "symbol": "TIME",
+    "decimals": 18,
+    "website": "https://meta-time.org/",
+    "description": "Time token is issued by Meta Time (Cayman)",
```

Comment:
description should be about your token

Reply:
description updated, please have a look, thanks